resources/js/visual-editor/blocks/separator/separator.css

```
@@ -0,0 +1,72 @@
+/*
+ * Separator — combined editor + frontend styles.
+ *
+ * Vendored from `@wordpress/block-library/src/separator/{style,editor,theme,deprecated}.scss`
+ * (v9.43.0). Sass converted to plain CSS; `data-type="core/separator"`
+ * selectors rewritten to `data-type="artisanpack/separator"`.
+ */
+
+/* style.scss */
+.wp-block-separator {
+    border-top: 2px solid currentColor;
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)